docs/sagas.md

```diff
@@ -0,0 +1,239 @@
+---
+id: sagas
+title: Sagas
+---
+
+A saga describes a long running process as a sequence of events.
+
+## Sagas Overview
+
+You can define a saga as a set of event handler functions. Each such function runs in response to a specific event and can do the following:
+
+- Send a command to an aggregate
+- Schedule a command on the specified moment in time
```

Comment:
Change "on the specified moment in time" to "at a specified moment in time".

Reply:
done